app_to_web.md

```diff
@@ -48,7 +48,17 @@ The existing API involves sending requests to the reporting origin to register e
 ```
 Attribution-Reporting-Support: os, web
 ```
-If this header indicates OS support, the reporting origin can optionally respond to the request with a [string structured header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3.3.3) that indicates a desire to use the OS’s attribution API instead of the browser’s. Note that the API also allows browsers to only support OS-level attribution if they choose.
+
+For subresource requests without the `Attribution-Reporting-Eligible` header,
+the server can optionally respond to the request with a [boolean structured header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3-3-6):
+```http
+Accept-Attribution-Reporting-Support: ?1
```

Comment:
What is the browser behavior is false is specified? While this header allows such a thing, I don't think it is particularly useful.

In #552, I tried to go with a design that somewhat aligns to client hints, where you denote what headers you want to receive, although that is potentially a bit heavy handed given we only have one supported header.

Do we have any other examples of APIs which use a boolean in this way?

Reply:
If false is specified then it’s the header will not be set.

Looks like topics API (Observe-Browsing-Topics header) uses a boolean this way as well (https://github.com/patcg-individual-drafts/topics#specific-details).

Reply:
Agree that it’s more flexible to have a generic Accept-Attribution-Reporting header that takes a list of headers, but given that currently we only have one header, I feel it’s a bit heavy handed. If we do have other headers to support in the future, we can consider adding one at that time. Wdyt?

Reply:
I think that sounds reasonable, and it is fairly similar to the topics use case.